plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/manager/SupportedApiVersionCommand.java

```diff
@@ -17,38 +17,42 @@
 // under the License.
 //
 
-package com.cloud.agent.api.guru;
+package com.cloud.agent.api.manager;
 
 import com.cloud.agent.api.Command;
 
-public class ReleaseVmVspCommand extends Command {
+public class SupportedApiVersionCommand extends Command {
 
-    String _networkUuid;
-    String _vmUuid;
-    String _vmInstanceName;
+    private final String _apiVersion;
 
-    public ReleaseVmVspCommand(String networkUuid, String vmUuid, String vmInstanceName) {
+    public SupportedApiVersionCommand(String apiVersion) {
         super();
-        this._networkUuid = networkUuid;
-        this._vmUuid = vmUuid;
-        this._vmInstanceName = vmInstanceName;
+        this._apiVersion = apiVersion;
     }
 
-    public String getNetworkUuid() {
-        return _networkUuid;
+    public String getApiVersion() {
+        return _apiVersion;
     }
 
-    public String getVmUuid() {
-        return _vmUuid;
+    @Override
+    public boolean executeInSequence() {
+        return false;
     }
 
-    public String getVmInstanceName() {
-        return _vmInstanceName;
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Implemented <code>equals</code> method for the <code>Command</code> super class, calling it from subclass.